src/main/java/com/hutchison/vibe/model/BotState.java

```diff
@@ -291,4 +320,10 @@ public void updateSavedQueue(String queueName, Long ownerId, MessageReceivedEven
             return Optional.of(event.getMember().getVoiceState().getChannel());
         }
     }
+
+    private String playingTrackText(Optional<String> currentTrackTitle) {
+        return currentTrackTitle.isPresent() ?
+                playingTrackText(currentTrackTitle) :
+                "No track playing.";
+    }
```

Comment:
@Clutchison Am I tripping or wouldn't this be an infinite loop? How is return type String without a .get? new fancy java 11 lol?